### PR TITLE
fix: Add a reference to rerouteToError annotation parameter

### PR DESCRIPTION
**Pull request by mshabarov**
Fixes https://github.com/vaadin/docs/issues/3013

### CODE REVIEW

File: articles/routing/exceptions.adoc [hunk 1]
```diff
@@ -210,6 +211,7 @@ public class BlogPost extends Component
 }
 
 @Tag(Tag.DIV)
+@PermitAll
```
Comment (mcollovati) [inline articles/routing/exceptions.adoc:214]: I think this should be `@AnonymousAllowed`. This error handler is not handling a security exception, so the error view should be public

File: articles/routing/exceptions.adoc [hunk 2]
```diff
@@ -227,11 +229,14 @@ public class FaultyBlogPostHandler extends Component
 }
 ----
 
+[interfacename]`HasErrorParameter` error view needs an access control annotation, so that Vaadin allows navigation to it. The example above uses [annotationname]`@PermitAll`, but [annotationname]`@RolesAllowed` can also be used. [annotationname]`@AnonymousAllowed` isn't recommended, as it exposes information about access restrictions to the anonymous users.
+
+Re-routing to a custom access denial views is also possible with annotation [annotationname]`@AccessDeniedErrorRouter`, see <<{articles}/advanced/custom-error-handler#, Customizing Error Messages for Unauthorized Views>>.
```
Comment (mcollovati) [inline articles/routing/exceptions.adoc:234]: Should this note maybe be moved to the previous paragraph?

**Review by mcollovati — CHANGES_REQUESTED** [review]

File: articles/routing/exceptions.adoc (hunk 1 above)
Comment (mshabarov) [reply articles/routing/exceptions.adoc:212]: Agree.

Comment (mshabarov) [reply articles/routing/exceptions.adoc]: Changed

File: articles/routing/exceptions.adoc (hunk 2 above)
Comment (mshabarov) [reply articles/routing/exceptions.adoc]: Yes, done.

**Review by mcollovati — APPROVED** [review]

**Pull request merged**